router.go

```diff
@@ -835,18 +835,19 @@ Admin:
 		statusColor := logs.ColorByStatus(iswin, statusCode)
 		methodColor := logs.ColorByMethod(iswin, r.Method)
 		resetColor := logs.ColorByMethod(iswin, "")
+		rip := utils.GetRequestIP(r)
```

Comment:
you can get the IP from `ctx.Input.IP()`. https://godoc.org/github.com/astaxie/beego/context#BeegoInput.IP